src/components/Donut/Donutchart.js

```diff
@@ -0,0 +1,70 @@
+import React, { useEffect, useState } from "react";
+import Chart from "react-apexcharts";
+import { ToastContainer, toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+import axios from "axios";
+
+export const Donutchart = () => {
+  const [mydata, setMydata] = useState([]);
+  const notify = () => toast("Error 404 not found DonutChart !!");
+
+  useEffect(() => {
+    axios
+      .get(
+        `https://raw.githubusercontent.com/Prakash-kumar1/JSON/main/testModel`
+      )
+      .then((response) => {
+        setMydata(response.data);
+      })
+      .catch(()=>{
+        {notify()} }) 
+  }, []);
+
+  const country = [];
+  const medal = [];
+  mydata.map((value) => {
```

Comment:
destructure value here

Reply:
Sir , i have used map outside return . so, there is no need to destructuring the value